Source/WebCore/platform/graphics/angle/GraphicsContextGLANGLE.cpp

```diff
@@ -572,6 +572,8 @@ void GraphicsContextGLANGLE::prepareTexture()
     if (contextAttributes().antialias)
         resolveMultisamplingIfNecessary();
 
+    GL_Finish();
+
```

Comment:
This one doesn't affect WPE but it doesn affect other ports, when it shouldn't.

Reply:
I supposed the bug was affecting other configurations as the cause is related to the synchronization of OpenGL previous calls before switching the FBO target texture.

Do you prefer that:
1- I enclose the modifications into a `#if USE(WPE).... #endif` to be only impacting WPE configuration, or
2- I validate that it is impacting other configurations ?

Reply:
I changed the approach to follow a more optimal way when fences are available.
I'm gonna upload a new version when I finish the changes and then I will ask for a new review.